action.yml

```diff
@@ -11,35 +11,61 @@ runs:
           source ${{ github.action_path }}/flag-alt-text.sh
 
           if [ ${{ github.event.comment }} ]; then
-            comment=$COMMENT
-            issue=${{ github.event.issue.html_url }}
-            comment_owner=${{ github.event.comment.user.login }}
-            is_pr=${{ github.event.issue.pull_request.url != '' }}
+            content=$COMMENT
```

Comment:
I renamed `comment` to `content` and `comment_owner` to `user`, since `comment` might be confusing given it's association with `github.event.comment`. We're setting these variables for non-comment events too.